core_v2/sources/v2_domains.move

```diff
@@ -108,23 +108,27 @@ module aptos_names_v2::v2_domains {
         token_addr: Option<address>,
     }
 
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
     /// Holder for `SetReverseLookupEvent` events
-    struct SetReverseLookupEvents has key, store {
+    struct SetReverseLookupEvents has key {
         set_reverse_lookup_events: event::EventHandle<SetReverseLookupEvent>,
     }
 
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
     /// Holder for `SetTargetAddressEvent` events
-    struct SetTargetAddressEvents has key, store {
+    struct SetTargetAddressEvents has key {
         set_name_events: event::EventHandle<SetTargetAddressEvent>,
     }
 
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
     /// Holder for `RegisterNameEvent` events
-    struct RegisterNameEvents has key, store {
+    struct RegisterNameEvents has key {
         register_name_events: event::EventHandle<RegisterNameEvent>,
     }
 
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
```

Comment:
probably should make a dedicated resource group for this... since it is all colocated within the same address....